keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListener.kt

```diff
@@ -78,14 +91,20 @@ class ArtifactListener(
    * Parses the status from a kork artifact, and throws an error if [releaseStatus] isn't
    * present in [metadata]
    */
-  private fun artifactStatus(artifact: Artifact): ArtifactStatus {
+  private fun debStatus(artifact: Artifact): ArtifactStatus {
     val status = artifact.metadata["releaseStatus"]?.toString()
       ?: throw IllegalStateException("Artifact event received without 'releaseStatus' field")
     return ArtifactStatus.valueOf(status)
   }
 
   private val artifactTypeNames by lazy { ArtifactType.values().map(ArtifactType::name) }
 
+  private fun Artifact.isDeb(): Boolean =
+    type.equals(DEB.toString(), true)
+
+  private fun Artifact.isDockerImage(): Boolean =
+    type.equals(DOCKER_IMAGE.toString(), true)
```

Comment:
I don’t understand this syntax. Isn’t it just `type == DOCKER_IMAGE`?

Reply:
This is on a `[korkArtifact](https://github.com/spinnaker/kork/blob/master/kork-artifacts/src/main/java/com/netflix/spinnaker/kork/artifacts/model/Artifact.java)`, not our artifact. So the type field is a string.

Reply:
Is that trailing `true` a default or something?

Reply:
It's ignore case = true

Reply:
ah